backend/src/routes/calendlyPage.ts

```diff
@@ -0,0 +1,62 @@
+import express, { NextFunction, Request, Response } from "express";
+
+const router = express.Router()
+
+// Add Firebase Auth
```

Comment:
Let's remove the calendly_link query argument from the route. Once we add auth to this, your route should get the mentor associated with this mentee through the pairing schema (honestly getting a mentee's mentor is a very common functionality so might be useful to put in a function if not already). Once this route has the mentorId, just use that to get the calendly_link for the mentor and populate this HTML file. 

This will make it simpler and nicer during integration. GOLDEN RULE - Always try to keep as much logic out of frontend and inside backend